components/compliance-service/reporting/relaxting/reports_test.go

```diff
@@ -206,3 +206,20 @@ func TestDoesControlTagMatchFilter(t *testing.T) {
 	multFilters["control_tag:shoe"] = []string{"blue"}
 	assert.Equal(t, true, doesControlTagMatchFilter(multFilters, tags2))
 }
+
+func TestFilterQuerychange(t *testing.T) {
```

Comment:
Success, failure and corner cases need to tested. This is not a good way of writing unit tests.